tests/integration/test_data_cli.py

```diff
@@ -218,6 +219,100 @@ def test_search_create_daily_email(invoke, search_result):
     assert json.loads(result.output) == search_result
 
 
+@respx.mock
+@pytest.mark.asyncio
+@pytest.mark.parametrize("filter", ['{1:1}', '{"foo"}'])
+@pytest.mark.parametrize(
+    "item_types", ['PSScene', 'SkySatScene', ('PSScene', 'SkySatScene')])
+def test_data_stats_invalid_filter(invoke, item_types, filter):
+    """Test for planet data search_create. Test with multiple item_types.
+    Test should fail as filter does not contain valid JSON."""
+    mock_resp = httpx.Response(HTTPStatus.OK,
+                               json={'features': [{
+                                   "key": "value"
+                               }]})
+    respx.post(TEST_STATS_URL).return_value = mock_resp
+    interval = "hour"
+    utc_offset = "+1h"
+    runner = CliRunner()
+    result = invoke(
+        ["stats", item_types, interval, filter, "--utc_offset", utc_offset],
+        runner=runner)
+    assert result.exit_code == 2
+
+
+@respx.mock
+@pytest.mark.parametrize(
+    "item_types", ['PSScene', 'SkySatScene', ('PSScene', 'SkySatScene')])
+@pytest.mark.parametrize("interval", ['hou', 'da', 'wek', 'moth', 'yr'])
+def test_data_stats_invalid_interval(invoke, item_types, interval):
+    """Test for planet data search_create. Test with multiple item_types.
+    Test should succeed as filter contains valid JSON."""
+    filter = {
+        "type": "DateRangeFilter",
+        "field_name": "acquired",
+        "config": {
+            "gt": "2019-12-31T00:00:00Z", "lte": "2020-01-31T00:00:00Z"
+        }
+    }
+
+    mock_resp = httpx.Response(HTTPStatus.OK,
+                               json={'features': [{
+                                   "key": "value"
+                               }]})
+    respx.post(TEST_STATS_URL).return_value = mock_resp
+
+    utc_offset = "+1h"
```

Comment:
we don't need to specify optional options that are unrelated to what is being tested.